src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -91,6 +91,40 @@ object SnowflakeTelemetry {
       case _ => None
     }
 
+  /**
+    * Put the pushdown failure telemetry message to internal cache.
+    * The message will be sent later in batch.
+    *
+    * @param plan The logical plan to include the unsupported operations
+    * @param exception The pushdown unsupported exception
+    */
+  def addPushdownFailMessage(plan: LogicalPlan,
+                             exception: SnowflakePushdownUnsupportedException)
+  : Unit = {
+    logger.info(
+      s"""Pushdown fails because of operation: ${exception.unsupportedOperation}
+         | message: ${exception.getMessage}
```

Comment:
You hardcoded this value, so it's always the same, right?

Reply:
I didn't get your question. what does "You hardcoded this value" refer to?